app/src/app/Docs.jsx

```diff
@@ -0,0 +1,212 @@
+// @flow
```

Comment:
Nice, now that it's in its own home, we can probs remove `DocsPage` from all these names (originally there to avoid conflicts/confusion). I can do this later if you prefer ofc.